src/inputs.ts

```diff
@@ -36,23 +37,23 @@ export const checkSpreadsheetStatus = async (): Promise<DirectomaticResponse> =>
  * @returns (Promise of RawRedirectProps[]) An array of raw redirect entries.
  */
 export const fetchRedirectRows = async (): Promise<RawRedirectProps[]> => {
-  return await fetch(lookup)
+  return fetch(lookup())
     .then((response) => response.json())
     .then((payload: any) => {
-      if (payload.values?.length < 0) {
+      if (payload.values?.length < 1) {
```

Comment:
Fix a bug where we don't declare a list empty when there are zero rows (as opposed to -1 rows)